src/app/modules/group/pages/group-edit/group-edit.component.ts

```diff
@@ -74,7 +76,8 @@ export class GroupEditComponent implements OnDestroy {
 
   getGroupChanges(): GroupChanges {
     return {
-      name: this.groupForm.get('name')?.value as string
+      name: this.groupForm.get('name')?.value as string,
+      description: this.groupForm.get('description')?.value as string,
```

Comment:
If the (trimmed) description entered is empty, it should be saved as "null".

Reply:
fixed